docs/src/pages/company/careers/careers.md

```diff
@@ -29,14 +29,35 @@ grow from it.
 
 We're currently looking for help in the following areas:
 
-### [Senior Software Engineer](/company/software-engineer/)
+### Engineering
 
-We are looking for senior software engineers to help support our open-source components, assist the Material-UI community, and grow our enterprise edition.
-
-### [Software Engineer Intern](/company/software-engineer-intern/)
+#### [Software Engineer Intern](/company/software-engineer-intern/)
 
 We're looking for software engineering intern to help support our open-source library, assist the Material-UI community, and help us build new components.
 
+## Future roles
+
+In the future, we will need to fill the following positions. If you don't want to wait for the positions to be opened, you can jump ahead and submit an [open application](https://airtable.com/shr9JdBSiE6noobhc).
+
+### Engineering
+
+- No enough visibility
+
+### Product
+
+- **Technical Product Manager**. You'll be combining customer feedback with the goals of the business to manage our roadmap and come up with the next killer feature. You'll work with our design, engineering and marketing teams to see those features through the entire development -> marketing cycle. You'll focus on measuring and driving adoption and conversion across our funnel.
+- **Developer Advocate**. Educate on the latest Material-UI features, craft high-quality examples and demos, engage with the community. Write documentation, and advocate for creating faster and more appealing UIs.
+- **Product Designer**. Design is critical to the success of our mission. We will be looking for skills that complement our Lead designer. It could be a graphics designer or a UX expert for instance, depending on our exact needs.
```

Comment:
```suggestion
- **Product Designer**. Design is critical to the success of our mission. We will be looking for skills that complement our lead designer. It could be a graphic designer or a UX expert for instance, depending on our exact needs.
```